random1on1/api/algorithm.py

```diff
@@ -0,0 +1,18 @@
+import abc
+import networkx
+import numpy
+
+
+class MatchingAlgorithm(metaclass=abc.ABCMeta):
```

Comment:
You should use `abc.ABC` for easiear readability. See [https://docs.python.org/3/whatsnew/3.4.html#abc](https://docs.python.org/3/whatsnew/3.4.html#abc).